app/static/src/app/store/appState/mutations.ts

```diff
@@ -21,11 +22,12 @@ export enum AppStateMutation {
 export const StateUploadMutations = [
     AppStateMutation.ClearQueuedStateUpload,
     AppStateMutation.SetQueuedStateUpload,
-    AppStateMutation.SetStateUploadInProgress
+    AppStateMutation.SetStateUploadInProgress,
+    AppStateMutation.SetPersisted
```

Comment:
We were getting a lot of unnecessary state uploads because of this not being included in the mutations to ignore as upload-worthy state changes..

Reply:
this is great! i had just thought it was intended or something but i did notice this!